### DISCUSSION TITLE
各位都用什么主题和登录方式

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3090] [讨论]
欧皇彩
正版

**Comment by MoYuan-CN** [comment]
正版：自定义主题

**Reply by SqrtTan** [reply]
我是个没有正版的可怜人……

**Comment by iPodtouch6557** [comment]
离线，自定义主题(有时会用活跃橙)
只要有了正版就功德圆满了:(
我另一台电脑没有内测版(大悲咒)

**Reply by Ad-closeNN** [reply]
如果是你的电脑，为什么不把PCL2内测版复制到那台电脑上
~~还是说不给这样做~~

**Reply by iPodtouch6557** [reply]
> 如果是你的电脑，为什么不把PCL2内测版复制到那台电脑上 ~还是说不给这样做~

给做，现在有了，而且比老电脑还发达

**Comment by kong-que-dnf123** [comment]
离线：自定义（有时为欧皇彩）
~我把自定义的渐变色拉的就特别像欧皇彩蚌埠住了~

**Comment by allMagicNB** [comment]
正版：秋仪金

**Comment by Deep-Dark-Forest** [comment]
Little skin（有正版但不用的屑，才不会告诉你是因为我的朋友没正版看不到我的皮肤）
主题：龙猫蓝用习惯了不想换，~~偶尔显摆一下会用活跃橙~~

**Comment by SuoHouDaiShi** [comment]
正版：自定义

**Comment by hejiehao** [comment]
跳票红
正版

**Comment by Chiloven945** [comment]
正版
自定义~版龙猫蓝~

**Reply by SqrtTan** [reply]
啊？

**Comment by wuliaodexiaoluo** [comment]
正版
活跃橙
